packages/dashmate/README.md

```diff
@@ -384,6 +384,46 @@ Then specify the created dotenv file as an option for the `docker-compose` comma
 ```bash
 $ docker-compose --env-file=.env.testnet up -d
 ```
+## Troubleshooting
+
+#### [FAILED] Node is not running
+One of your nodes failed to start or missing, you may retry with --force option
+
+`dashmate stop --force`
+
```

Comment:
We need to explain that for group of nodes (local), `dashmate group stop --force` needs to be used